draft/spec/change-log.md

```diff
@@ -0,0 +1,108 @@
+
+#### Clarify manifest block requirements
+
+Add language in [Manifest](https://ocfl.io/draft/spec/#manifest) section to clarify that the `manifest` block must be a JSON object (adding error code [E106](https://ocfl.io/draft/spec/#E106)) and that the each key must correspond to a digest value key found in one or more `state` blocks (adding error code [E107](https://ocfl.io/draft/spec/#E107)). (Issue [#537](https://github.com/OCFL/spec/issues/537))
+
+#### Clarify manifest requirements in historic inventories
+
+Wording of the [Content Directory](https://ocfl.io/draft/spec/#content-directory) section improved to make it clear that for each historical inventory, the manifest must reference every file in that version directory. (Issue [#538](https://github.com/OCFL/spec/issues/538))
+
+#### Clarify language and error codes for version numbers
+
+Change [Version Directories](https://ocfl.io/draft/spec/#version-directories) section changed to be more specific about version numbers. Adds error code [E104](https://ocfl.io/draft/spec/#E104) for the specific case of missing prefix `v`, and [E105](https://ocfl.io/draft/spec/#E105) for the specific case of using positive base-ten integers. (Issue [#532](https://github.com/OCFL/spec/issues/532))
+
+#### Clarify that the content directory must be a direct child of the version directory
+
+Change [Content Directory](https://ocfl.io/draft/spec/#content-directory) section to make it clear that the `contentDirectory` must indicate a direct child of the version directory. Adds error code [E108](https://ocfl.io/draft/spec/#E108). (Issue [#530](https://github.com/OCFL/spec/issues/530))
+
+#### Clarify that `id` must be the same across all versions
+
+Update [Basic Structure](https://ocfl.io/draft/spec/#inventory-structure) section to make it clear that the `id` must not change between versions of the same object. Adds error code [E110](https://ocfl.io/draft/spec/#E110). (Issue [#542](https://github.com/OCFL/spec/issues/542))
+
+#### Use logical state consistently
+
+Use the notion of "logical state" consistently in the [Version](https://ocfl.io/draft/spec/#version), [Version Inventory and Inventory Digest](https://ocfl.io/draft/spec/#version-inventory) and [BagIt in an OCFL Object](https://ocfl.io/draft/spec/#example-bagit-in-ocfl) sections. (Issue [#571](https://github.com/OCFL/spec/issues/571))
+
+#### Clarify digest value case sensitivity requirements
+
+Change [Manifest](https://ocfl.io/draft/spec/#manifest) and [Fixity](https://ocfl.io/draft/spec/#fixity) sections to make it clear that the additional requirement for each digest value to appear only once in the manifest or fixity block applies only in the case of case-insensitive digest algorithms. (Issue [#573](https://github.com/OCFL/spec/issues/573))
```

Comment:
Suggestion from:
```
applies only in the case of case-insensitive digest algorithms
```
to:
```
applies only to case-insensitive digest algorithms
```

Reply:
Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)